app/javascript/components/upload/WizardNavPanel.jsx

```diff
@@ -176,6 +170,52 @@ function clusteringHelpContent(isClusteringParsing) {
   </Popover>
 }
 
+/**
+ * Return the appropriate display of the main steps based on traditional or AnnData upload experience
+ */
+function mainStepsDisplay(formState, serverState, currentStep, setCurrentStep, mainSteps,
+  isAnnDataExperience, annDataMainExpanded, setAnnDataMainExpanded, expansionIcon) {
+  if (isAnnDataExperience) {
+    return <ul className="upload-wizard-steps" role="tablist" data-analytics-name="upload-wizard-anndata-primary-steps">
```

Comment:
Let's align names between analytics and implementation, unless there's a compelling reason not to.

```suggestion
    return <ul className="upload-wizard-steps" role="tablist" data-analytics-name="upload-wizard-anndata-main-steps">
```
```suggestion
    return <ul className="upload-wizard-steps" role="tablist" data-analytics-name="upload-wizard-anndata-primary-steps">
```

Reply:
I named this based on the existing analytics name which is "upload-wizard-primary-steps", do you want me to change the existing name (for traditional view) and then make the AnnData version match so that both use 'main' rather than 'primary' ?

Reply:
NVM I see you already answered this in a different comment